### DISCUSSION TITLE
10 sec delay

### DISCUSSION THREAD

**Discussion by sagitta** [briis/unifiprotect#325] [Q&A] [answered]
Hi,

First, thank you for the work done on this plugin by the author and all that have helped.

One thing i noted is the fact that there is a 10 seconds delay on the stream of home assistant whereas on the rstp stream played on VLC I don't have this delay.

Do you know what should I do in order to reduce/cancel this delay?

Thank you in advance

**Comment by AngellusMortis** [comment, marked as answer]
This is not possible for something us to solve with only a custom component. Ideally it would require proper support from core to able to stream RTSP to something that is _not_ HLS, which is what is introducing the delay. See this [feature request I made](https://community.home-assistant.io/t/webrtc-or-lhls-support-for-camera-stream-components/344909) with more details.

**Reply by sagitta** [reply]
2 minutes to get the answer !

I don't remember last time I got the correct answer after such little time :)

Thank you for the explaination, I'll go on the thread you created on hass community.
